larg4/Analysis/CheckMCParticle_module.cc

```diff
@@ -56,19 +56,43 @@ void larg4::CheckMCParticle::beginJob()
 
 void larg4::CheckMCParticle::analyze(const art::Event& event)
 {
+  auto allDropped =  event.getMany<std::set<int>>();
+  for (auto const& dropped : allDropped) {
+    
+    std::cout << "List of dropped Tracks: "<<std::endl;
+    for(auto const& droppedTrack :  *dropped )
+      {
+	std::cout << droppedTrack << " ";
+      }
+    std::cout <<std::endl;   
+  }
```

Comment:
Suggest wrapping this printout in an `#if defined _verbose_` block, similar to what is done below.

Reply:
Hi Kyle, 
done. 
thanks for the comments. Should have said that I am planning for a cleaned up version for next week.